--- pcmanfm-0.9.8/configure.ac~	2010-10-03 21:17:26.000000000 +0200
+++ pcmanfm-0.9.8/configure.ac	2010-12-06 09:53:30.253743000 +0100
@@ -3,7 +3,7 @@
 
 AC_PREREQ([2.63])
 AC_INIT([pcmanfm], [0.9.8], [http://pcmanfm.sf.net/])
-AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_INIT_AUTOMAKE([foreign])
 
 AC_CONFIG_HEADERS([config.h])
 
--- pcmanfm-0.9.8/src/Makefile.am~	2010-09-23 13:21:03.000000000 +0200
+++ pcmanfm-0.9.8/src/Makefile.am	2010-12-06 09:54:05.637076334 +0100
@@ -32,8 +32,6 @@
 	$(FM_CFLAGS) \
 	$(MENU_CACHE_CFLAGS) \
 	$(G_CAST_CHECKS) \
-	-Wall \
-	-Werror-implicit-function-declaration \
 	$(NULL)
 
 pcmanfm_LDADD = \
